pkg/meta/common/common.go

```diff
@@ -286,3 +287,98 @@ func CheckImageLastUpdated(repoLastUpdated time.Time, isSigned bool, noImageChec
 
 	return repoLastUpdated, noImageChecked, isSigned
 }
+
+func FilterFoundData(foundRepos []repodb.RepoMetadata, manifestMetadataMap map[string]repodb.ManifestMetadata,
```

Comment:
I think this needs a better name, "found" is about the context where the function is used, and it doesn't "Filter" doesn't by an obvious criteria.

Maybe, if I understand correctly:
```suggestion
func FilterDataByRepo(foundRepos []repodb.RepoMetadata, manifestMetadataMap map[string]repodb.ManifestMetadata,
```

Reply:
Yes I think that's a better name for this function